packages/material-ui-lab/src/TimelineSeparator/TimelineSeparator.test.js

```diff
@@ -1,20 +1,19 @@
 import * as React from 'react';
-import { getClasses, createMount, describeConformance } from 'test/utils';
+import { createClientRender, createMount, describeConformanceV5 } from 'test/utils';
 import TimelineSeparator from './TimelineSeparator';
+import classes from './timelineSeparatorClasses';
```

Comment:
For the future, I would recommend we always test the public API

```suggestion
import TimelineSeparator, { timelineSeparatorClasses as classes } from '@material-ui/lab/TimelineSeparator';
```

Reply:
Done in #25682